cfgmgr/natmgrd.cpp

```diff
@@ -197,10 +206,14 @@ int main(int argc, char **argv)
             auto *c = (Executor *)sel;
             c->execute();
         }
+
+        cleanup();
```

Comment:
>cleanup [](http://example.com/codeflow?start=8&length=7)

Call `cleanup()` before every `return`, even inside `catch`. So better move this line to outside `try`, and just above bottom `return 0`.